_posts/0000-01-02-jezuma.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![image](https://user-images.githubusercontent.com/67762344/86431992-808b4480-bcb3-11ea-8571-590473eb99f3.png)
This is funny 👍